File: April 10, 2026.js
function rookAttack(rook1, rook2) {

  const letterRook1 = rook1[0];
  const letterRook2 = rook2[0];
  const nRook1 = rook1.slice(1);
  const nRook2 = rook2.slice(1);

  return letterRook1 == letterRook2 || nRook1 == nRook2;
}

File: April 10, 2026.py
def rook_attack(rook1, rook2):

    letter_rook1 = rook1[0]
    letter_rook2 = rook2[0]
    n_rook1 = rook1[1:]
    n_rook2 = rook2[1:]
    
    return letter_rook1 == letter_rook2 or n_rook1 == n_rook2